packages/react-components/react-provider/src/components/FluentProvider/useFluentProviderThemeStyleTag.ts

```diff
@@ -1,63 +1,68 @@
-import { useId, usePrevious } from '@fluentui/react-utilities';
+import { useId } from '@fluentui/react-utilities';
 import * as React from 'react';
 import type { FluentProviderState } from './FluentProvider.types';
 import { fluentProviderClassNames } from './useFluentProviderStyles';
 
+const useInsertionEffect = (React as never)['useInsertion' + 'Effect']
+  ? (React as never)['useInsertion' + 'Effect']
+  : // eslint-disable-next-line no-restricted-properties
+    React.useLayoutEffect;
```

Comment:
```suggestion
    useIsomorphicLayoutEffect;
```
Otherwise this will throw on SSR